### PR TITLE
Mod YAML troubleshooting updates

### PR DESCRIPTION
**Pull request by benny-dreamly**

### CODE REVIEW

File: Documentation/Supported Mods.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ To include supported mods in your multiworld slot, you need to include a section
 This section must be an array with the **exact** names of every mod you wish to include. Any improperly typed mod name will be ignored.
 ![image](https://i.imgur.com/uOHtXmU.png)
 
+If manually typing your mods list isn't working, try going to the [weighted settings page](https://archipelago.gg/weighted-options) and selecting stardew and then leaving all the other settings and check off the boxes for the mods you want. Export the settings, and copy paste the result into your own yaml file.
```
Comment (agilbert1412) [inline Documentation/Supported Mods.md:54]: Could we try to type this in a more "formal" tone, like the rest of the document? "Try going to the weighted settings" feels like a shift in tone and less official than the surrounding stuff

**Review by agilbert1412 — CHANGES_REQUESTED** [review]

File: Documentation/Supported Mods.md (hunk 1 above)
Comment (benny-dreamly) [reply Documentation/Supported Mods.md:54]: Sure! Let me do some thinking.

Comment (benny-dreamly) [reply Documentation/Supported Mods.md]: does this sound good? 
Screenshot 2024-01-29 at 6 30 55 PM

Comment (benny-dreamly) [reply Documentation/Supported Mods.md]: I just noticed a missing comma... oops

File: Documentation/Supported Mods.md [hunk 2]
```diff
@@ -51,6 +51,8 @@ To include supported mods in your multiworld slot, you need to include a section
...
 
+Alternatively, the [weighted settings page](https://archipelago.gg/weighted-options) is another option available to add mods to your yaml file. First, click the link. In the section titled **Game Select**, drag the slider beside **Stardew Valley** to 1. Next, scroll down to the secton titled **Mods**. There, you will find a series of checkboxes. Check the checkboxes beside the name of the mods that you'd like to include in your yaml file. Lastly, click the **export settings** button at the bottom of the page, which will download the yaml file to your computer. Open that yaml file in your preferred text editor, and look for the section titled **Mods**. Select that entire section, and copy it to your clipboard. Then, if you haven't already done so, open your yaml file that you'd like to include the mods in, and paste what you've just copied over into it. Make sure that you select over the pre-existing mods category that is included by default by the website. This is the most important step. Ensure you **save** the file to wherever you'd like to save it to. For good measure, open the saved file again to make sure that it saved properly. You've now added mods to your yaml file!
```
Comment (agilbert1412) [inline Documentation/Supported Mods.md:54]: I think this whole thing would be worth making a new section, right above ## Bug Reports. Maybe like
## Having trouble with the yaml syntax?
blablabla

**Review by agilbert1412 — CHANGES_REQUESTED** [review]

File: Documentation/Supported Mods.md (hunk 2 above)
Comment (benny-dreamly) [reply Documentation/Supported Mods.md]: sounds good.

**Review by agilbert1412 — APPROVED** [review]

**Pull request merged**